### PR TITLE
Moving get and set gNMI responses to new atomix based workflow

### PR DESCRIPTION
**Pull request by Andrea-Campanella**
THis PR moves the gNMI get and set methods to the atomix based workflow.

TODO list for reference at this point in time:
- Line 353 method checkForReadOnly, used at line 103
- ExtractTypeAndVersion line 411 of manager.go
- Lines 130 and 157 of set.go need to be update after finishing of ExtractTypeAndVersion
- make sure Line 91 of setconfig.go works with update ExtractTypeAndVersion
- RollbackConfig line 60 make sure it’s the last applied one
- Change Description line 383
- Line 172 of setconfig.go Check User

### CODE REVIEW

File: pkg/northbound/gnmi/get.go [hunk 1]
```diff
@@ -164,6 +164,7 @@ func getUpdate(prefix *gnmi.Path, path *gnmi.Path) (*gnmi.Update, error) {
 		pathAsString = utils.StrPath(prefix) + pathAsString
 	}
 	//TODO the following can be optimized by looking if the path is in the read only
+	//Currently un-hooked from the response, just generating values for comparison
 	configValues, err := manager.GetManager().GetTargetConfig(target, "",
```
Comment (SeanCondon) [inline pkg/northbound/gnmi/get.go]: i'd rename this to configValuesOld, and put deprecated before it

File: pkg/northbound/gnmi/set.go [hunk 2]
```diff
@@ -248,10 +252,10 @@ func (s *Server) Set(ctx context.Context, req *gnmi.SetRequest) (*gnmi.SetRespon
 		Extension: extensions,
 	}
 
-	log.Info("set response ", setResponse)
-	log.Info("atomix update response ", setResponseAtomix)
+	log.Info("UNUSED - set response ", setResponse)
+	log.Info("USED - atomix update response ", setResponseAtomix)
 
-	return setResponse, nil
```
Comment (SeanCondon) [inline pkg/northbound/gnmi/set.go]: where ever it is created, I would rename `setResponse` to `setResponseOld`

**Comment by Andrea-Campanella** [comment]
@SeanCondon this version of the patch is ready for work to make the models test pass.

**Comment by Andrea-Campanella** [comment]
this patch now passes all e2e integrations tests

**Comment by Andrea-Campanella** [comment]
last element that needs to be passed to new stores is
setconfig.go line 103 checkForReadOnly method
@ray-milkey if you want it's ready for manager tests.

**Comment by ray-milkey** [comment]
I am going to force merge this and then skip the tests in another PR. I can't get git to behave and I want these changes in.

**Pull request merged**